src/website/views/components/dropdown/_properties.pug

```diff
@@ -1,7 +1,151 @@
 h2 JavaScript
-h3 Properties
+h3.-mt--2 Properties
```

Comment:
@soujanreddy Web component Properties are compiled automatically adding `:stencilDoc(name='chi-dropdown')`

Reply:
I see we don't have a data object set for chi-dropdown in docs.json. I might have to add that object first to use :stencilDoc(name='chi-dropdown')